src/shared/seccomp-util.c

```diff
@@ -132,6 +137,14 @@ const char* seccomp_arch_to_string(uint32_t c) {
                 return "mips64-le";
         case SCMP_ARCH_MIPSEL64N32:
                 return "mips64-le-n32";
+#ifdef SCMP_ARCH_PARISC
```

Comment:
I've opted to keep the #ifdef guards but they're trivial to remove. Support for HPPA landed in libseccomp in 2016 but was released in libseccomp 2.4.0 (2019).

Reply:
We have similar ifdef for `SCMP_ARCH_RISCV64`. So, that's OK. But please add similar `#ifdef` at all places where `SCMP_ARCH_PARISC` or friends are used.